src/events/crawler/scrapers/AUS/NSW/index.js

```diff
@@ -0,0 +1,32 @@
+import * as parse from '../../../lib/parse.js';
+import * as fetch from '../../../lib/fetch.js';
+
+/**
+ * @param {Cheerio} $row
+ */
+const parseRow = $row => parse.number($row.find('td:last-child').text());
+
+const scraper = {
+  country: 'AUS',
+  state: 'New South Wales',
+  url:
+    'https://www.health.nsw.gov.au/_layouts/feed.aspx?xsl=1&web=/news&page=4ac47e14-04a9-4016-b501-65a23280e841&wp=baabf81e-a904-44f1-8d59-5f6d56519965&pageurl=/news/Pages/rss-nsw-health.aspx',
+  type: 'table',
```

Comment:
Can you add a `sources` array with the human-readable name of the organization and the human-readable webpage with the same info? Deets on the properties here: https://github.com/lazd/coronadatascraper#writing-a-scraper

Also, add yourself as a maintainer! Put an object in `lib/maintainers.js` and import that wherever you're maintaining `maintainers: [maintainers.camjc]`

Thanks!